src/components/Common/Misc/ContentSelector/ContentSelector.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This function is responsible for rendering the content of the Agent Issuer,
+ * Agent Holder, and Agent Verifier pages based on value of persona
+ * @returns The ContentWrap component.
+ */
```

Comment:
Not the case anymore?

Reply:
I guess all comments are no longer relevant?